native/winp.sln

```diff
@@ -15,32 +15,48 @@ Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Win32 = Debug|Win32
 		Debug|x64 = Debug|x64
+		Debug|am64 = Debug|am64

```

Comment:
Should this be "arm64" rather than "am64"?  I am not skilled user of Visual Studio solution files, so am not sure if the value should use the same string as the release configuration or if it is an intentional difference.  I checked the [Visual Studio documentation](https://learn.microsoft.com/en-us/visualstudio/ide/understanding-build-configurations?view=vs-2019) and did not find a list of allowed values.
```suggestion
		Debug|am64 = Debug|am64

```

Reply:
Good catch. `sln` files are used to open project in VS interface, but not from the build scripts we have in this repo. It still needs to be fixed, thanks!